manifests/desktop.pp

```diff
@@ -37,6 +41,12 @@
     }
   }
 
+  exec { 'dconf update':
+    command     => '/usr/bin/dconf update',
```

Comment:
Exec commands should have acceptance test coverage as they historically have a high breakage rate.

Reply:
Punted to simp/dconf